tool/tsh/tsh.go

```diff
@@ -336,6 +351,10 @@ const (
 	proxyDefaultResolutionTimeout = 2 * time.Second
 
 	teleportNamespace = "teleport.dev"
+
+	// .tsh config must go in a subdir as all .yaml files in .tsh get
+	// loaded automatically
```

Comment:
Could you elaborate what the problem is with storing it in just `~/.tsh/config.yaml`?